rest_api/src/test/java/tqs/cloudit/services/AuthenticationStepsDefs.java

```diff
@@ -4,30 +4,28 @@
 import cucumber.api.java.en.When;
 import cucumber.api.java.en.Then;
 import java.util.TreeSet;
-import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import org.junit.Ignore;
-import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.WebDriverWait;
 import org.springframework.beans.factory.annotation.Autowired;
 import tqs.cloudit.domain.rest.User;
+import org.openqa.selenium.phantomjs.PhantomJSDriver;
 
 @Ignore
 public class AuthenticationStepsDefs extends TestApplication {
     
-    private final WebDriver driver = new FirefoxDriver();
+    private final WebDriver driver = new PhantomJSDriver();
```

Comment:
Because the tests are ran in a docker container and the binary driver is necessary we need to download it first.
See [example](https://github.com/aspedrosa/CloudIT/blob/master/rest_api/src/test/java/tqs/cloudit/services/StepsDefs.java#L26). And you have to include [this](https://github.com/aspedrosa/CloudIT/blob/master/rest_api/pom.xml#L133)